[ci:docs] Remove future tense from man pages (#18559)
containers/podman: docs/source/markdown/podman.1.md

For the netavark backend "/etc/containers/networks" is used as root
and "$graphroot/networks" as rootless.
For the CNI backend the default is "/etc/cni/net.d" as root

> and "$HOME/.config/cni/net.d" as rootless. CNI is deprecated from Podman in the future, use netavark.

Reviewer: ```suggestion
and "$HOME/.config/cni/net.d" as rootless. CNI will be deprecated from Podman in the future, use netavark.
```

#### **--noout**

> Redirect stdout to /dev/null. This command prevents all stdout from the Podman command. The **--noout**  option is not block stderr or stdout from containers.

Reviewer: ```suggestion
Redirect stdout to /dev/null. This command prevents all stdout from the Podman command. The **--noout**  option does not block stderr or stdout from containers.
```

for cgroup V2, the default runtime is `crun`, the manpage to consult is `crun(8)`.).

Note: Do not pass the leading `--` to the flag. To pass the runc flag `--log-format json`

> to podman build, the option given can be `--runtime-flag log-format=json`.

Reviewer: ```suggestion
to `podman build`, the option given can be `--runtime-flag log-format=json`.
```

Storage driver.  The default storage driver for UID 0 is configured in `containers-storage.conf(5)` in rootless mode), and is *vfs* for non-root users when *fuse-overlayfs* is not available.  The `STORAGE_DRIVER` environment variable overrides the default.  The --storage-driver specified driver overrides all.

Overriding this option causes the *storage-opt* settings in `containers-storage.conf(5)` to be ignored.  The user must
specify additional options via the `--storage-opt` flag.

#### **--storage-opt**=*value*

> Specify a storage driver option. Default storage driver options are configured in `containers-storage.conf(5)`. The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options override all. Specify --storage-opt="" so no storage options is used.

Reviewer: ```suggestion
Specify a storage driver option. Default storage driver options are configured in `containers-storage.conf(5)`. The `STORAGE_OPTS` environment variable overrides the default. The --storage-opt specified options override all. Specify --storage-opt="" so no storage options are used.
```

The mounts.conf file specifies volume mount directories that are automatically mounted inside containers when executing the `podman run` or `podman start` commands. Administrators can override the defaults file by creating `/etc/containers/mounts.conf`.

When Podman runs in rootless mode, the file `$HOME/.config/containers/mounts.conf` overrides the default if it exists. Please refer to containers-mounts.conf(5) for further details.

**policy.json** (`/etc/containers/policy.json`)

Signature verification policy files are used to specify policy, e.g. trusted keys, applicable when deciding whether to accept an image, or individual signatures of that image, as valid.

**registries.conf** (`/etc/containers/registries.conf`, `$HOME/.config/containers/registries.conf`)

> registries.conf is the configuration file which specifies which container registries is consulted when completing image names which do not include a registry or domain portion.

Reviewer: ```suggestion
registries.conf is the configuration file which specifies which container registries are consulted when completing image names that do not include a registry or domain portion.
```
